WordPress/src/main/java/org/wordpress/android/util/WPLegacyMigrationUtils.java

```diff
@@ -144,10 +145,10 @@ public static boolean hasSelfHostedSiteToMigrate(Context context) {
         }
     }
 
-    private static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
+    static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
```

Comment:
I found one thing we should do here (or maybe change the default value in the SiteModel in FluxC). We must set the isVisible flag to true, self hosted sites are always visible. Also I'm not sure we update that flag for self hosted on site fetched in FluxC.